LiteFarmOrg/LiteFarm
# Added date to emails

This PR adds a date to the subject line of the support emails (see ticket here: https://lite-farm.atlassian.net/browse/LF-2362)

To test:
1. Click on the Profile icon in the top right-hand corner
2. Click on the Help button
3. Submit a help request email
4. You should have an email in your inbox with the date and time included in the subject line
